src/app/proto/validator/accounts/v2/web_api.ts

```diff
@@ -216,3 +216,13 @@ export interface RecoverWalletRequest {
    */
   wallet_password: string;
 }
+export interface AccountVoluntaryExitRequest {
+  /**
+   * The key from the accounts that will be removed
+   */
+  publicKeys: string[];
+  /**
+   * Confirmation if the exit should proceed or not
+   */
+  confrimation: string;
```

Comment:
I don't think `confrimation` is needed at all. Once we submit the request, it will always be `true` (it doesn't make sense to submit a request when it's `false`).

Reply:
I will remove it from the code i thought maybe you would double-check on the back end.